pkg/cmd/certinspection/certinspection.go

```diff
@@ -98,18 +99,29 @@ func (o *CertInspectionOptions) Validate() error {
 func (o *CertInspectionOptions) Run() error {
 	visitor := o.resourceFinder.Do()
 
+	inClusterResourceData := &certgraphapi.PerInClusterResourceData{}
+	certs := []*certgraphapi.CertKeyPair{}
+	caBundles := []*certgraphapi.CertificateAuthorityBundle{}
+
 	err := visitor.Visit(func(info *resource.Info, err error) error {
 		if err != nil {
 			return err
 		}
 
 		switch castObj := info.Object.(type) {
 		case *corev1.ConfigMap:
-			inspectConfigMap(castObj)
+			if details := inspectConfigMap(castObj); details != nil {
+				caBundles = append(caBundles, details)
+			}
 		case *corev1.Secret:
-			inspectSecret(castObj)
-		// case *certificatesv1beta1.CertificateSigningRequest:
-		// 	inspectCSR(castObj)
+			details := inspectSecret(castObj)
+			for _, detail := range details {
+				if detail != nil {
+					certs = append(certs, detail)
+				}
+			}
+		case *certificatesv1beta1.CertificateSigningRequest:
+			inspectCSR(castObj)
 		default:
 			return fmt.Errorf("unhandled resource: %T", castObj)
```

Comment:
Missing Error Context  here.
```suggestion
			return fmt.Errorf("unhandled resource: %T (%s/%s)", castObj, info.Namespace, info.Name)

```